integration/src/test/java/org/kie/hacep/PodAsLeaderTest.java

```diff
@@ -72,6 +75,18 @@ public void tearDown() {
         kafkaServerTest.shutdownServer();
     }
 
+    private EnvConfig getEnvConfig() {
```

Comment:
This code is replicated in more tests. It is at least present in "PodAsReplicaTest". Extract it from tests and share it. It seems to me that the configuration of the most of the tests is same. Please put it into common parent or util methods.

Reply:
ok